### PR TITLE
Integrated HA remote nodes into the UI

### PR DESCRIPTION
**Pull request by tboerger**
There are still missing bits on the backend to retrieve the correct roles. The frontend seem to work fine, including the contraints. So my task is done, the rest should be up to another team.

### CODE REVIEW

**Comment by jsuchome** [comment]
Could you add some screenshot(s)?

**Comment by tboerger** [comment]
screenshot from 2016-01-19 15-50-55
screenshot from 2016-01-19 15-51-44

**Comment by tboerger** [comment]
> Could you add some screenshot(s)?

Directly while taking screenshots to upload :dancers:

**Comment by tboerger** [comment]
Note: Ignore the hound issues, it's a bug within HoundCI which ignores the excludes.

File: crowbar_framework/config/locales/crowbar/en.yml [hunk 1]
```diff
@@ -137,6 +137,9 @@ en:
       delete_proposal_failure: 'Failed to delete the proposal'
       confirm_delete_proposal: 'Are you sure you wish to delete this proposal?'
       available_clusters: 'Available Clusters'
+      available_clusters_hint: 'Here you can see all available clusters that do not have any additional remote nodes.'
+      available_remotes: 'Available Remotes'
+      available_remotes_hint: 'Here you can see all available clusters that do have additional remote nodes assigned.'
```
Comment (jsuchome) [inline crowbar_framework/config/locales/crowbar/en.yml]: *do have* or just *have* ?

**Comment by tboerger** [comment]
**Depends on https://github.com/crowbar/crowbar-core/pull/193**

**Comment by vuntz** [comment]
@tboerger thanks for the PR; I did a few changes and submitted https://github.com/SUSE-Cloud/crowbar-core/pull/7. Could you check this and merge if that's fine?

**Comment by tboerger** [comment]
@vuntz as I'm at home I have just merged and updated this PR so that you can make further progress on that.

**Comment by vuntz** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Comment by vuntz** [comment]
FWIW, gating fails because the patch contains new binary files (fonts), and so we can't build an updated package.

**Comment by tboerger** [comment]
So merge this?

**Comment by vuntz** [comment]
Two +1 and the PR we depended on is merged => merging (as a reminder: gating can't pass due to binary files; we used manual testing instead)

**Pull request merged**